src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -24,7 +24,7 @@ interface MetricsEnabled {
 export interface ServerName {
```

Comment:
Oh wow, I didn't notice that. Good catch.

In that case, we shouldn't be exporting any of the types or interfaces defined here or using them in the ShadowboxServer public interface.

It seems like `ServerName` and `MetricsEnabled` aren't being used at all in the codebase. Can we remove those (or not export `ServerName` at least)?

Reply:
Removed both.